src/data/widgets.js

```diff
@@ -28,24 +28,24 @@ const TestnetWidgets = {
 
 const MainnetWidgets = {
   image: "mob.near/widget/Image",
-  default: "adminalpha.near/widget/ActivityPage",
+  default: "near/widget/ActivityPage",
   viewSource: "mob.near/widget/WidgetSource",
   widgetMetadataEditor: "mob.near/widget/WidgetMetadataEditor",
   widgetMetadata: "mob.near/widget/WidgetMetadata",
   profileImage: "mob.near/widget/ProfileImage",
   profileName: "patrick.near/widget/ProfileName",
   editorComponentSearch: "mob.near/widget/Editor.ComponentSearch",
   profileInlineBlock: "mob.near/widget/Profile.InlineBlock",
-  componentsPage: "adminalpha.near/widget/ComponentsPage",
-  peoplePage: "adminalpha.near/widget/PeoplePage",
+  componentsPage: "near/widget/ComponentsPage",
+  peoplePage: "near/widget/PeoplePage",
   globalSearchPage: "chaotictempest.near/widget/Search",
-  notificationButton: "adminalpha.near/widget/NotificationButton",
-  profilePage: "adminalpha.near/widget/ProfilePage",
-  componentSummary: "adminalpha.near/widget/ComponentSummary",
-  notificationsPage: "adminalpha.near/widget/NotificationsPage",
-  tosCheck: "adminalpha.near/widget/TosCheck",
+  notificationButton: "near/widget/NotificationButton",
+  profilePage: "near/widget/ProfilePage",
+  componentSummary: "near/widget/ComponentSummary",
+  notificationsPage: "near/widget/NotificationsPage",
+  tosCheck: "near/widget/TosCheck",
   tosContent: "adminalpha.near/widget/TosContent",
```

Comment:
`tosContent` has to stay as `adminalpha.near` for now or all users will be prompted to re-accept TOS